components/BottomNavigation/examples/BottomNavigationTitleVisibilityChangeExample.swift

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import MaterialComponents.MaterialBottomNavigation_ColorThemer
+import MaterialComponents.MaterialColorScheme
+
+class BottomNavigationTitleVisibilityChangeExample: UIViewController, MDCBottomNavigationBarDelegate {
+  
+  var colorScheme = MDCSemanticColorScheme()
+  
+  // Create a bottom navigation bar to add to a view.
+  let bottomNavBar = MDCBottomNavigationBar()
+  
+  init() {
+    super.init(nibName: nil, bundle: nil)
+    commonBottomNavigationTypicalUseSwiftExampleInit()
+  }
+  
+  @available(*, unavailable)
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+  
+  func commonBottomNavigationTypicalUseSwiftExampleInit() {
+    view.backgroundColor = .lightGray
```

Comment:
Nit: Please use `colorScheme.backgroundColor` for the background color.

Reply:
Fixed, here and in other BottomNav Swift examples.